crates/biome_analyze/src/rule.rs

```diff
@@ -157,6 +157,10 @@ pub enum RuleSource {
     EslintVitest(&'static str),
     /// Rules from [Eslint Plugin Vue.js](https://eslint.vuejs.org/)
     EslintVueJs(&'static str),
+    /// Rules from [Eslint Plugin Package.json](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json)
```

Comment:
the official package name is `eslint-plugin-package-json`?
```suggestion
    /// Rules from [eslint-plugin-package-json](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json)
```

Reply:
As dyc3 already mentioned, that doesn't match the pattern of previous lines